.gitignore

```diff
@@ -1,5 +1,6 @@
 # Created by https://www.gitignore.io
 
+src/main/java/j2html/Experiment.java
```

Comment:
This seems like something that shouldn't be tracked.

Reply:
yeah, i forgot i used that one for trying out stuff.